src/Lib/Mail/Mail.php

```diff
@@ -0,0 +1,105 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Lib\Mail;
+
+class Mail
+{
+
+    /** @var string */
+    private $headders;
+
+
+    /**
+     *
+     * @param string $to
+     * @param string $from
+     * @param string $fromName
+     * @param string $content
+     */
+    public function __construct(string $to, string $from, string $fromName, string $subject, string $content)
+    {
+        $this->to = $to?$to:false;
+        $this->from = testEmail($from, 'From')?$from:false;
+        $this->fromName = $fromName?$fromName:false;
+        $this->subject = $subject?$subject:false;
+        $this->content = $content?$content:false;
+    }
+
+
+    /**
+    * Add hidden copy
+    *
+    * @return void
+    */
+    public static function addBcc($bcc):void
+    {
+        $this->bcc = testEmail($bcc, 'Bcc')?$bcc:false;
+    }
+
+
+    /**
+    * Add reply to (different address)
+    *
+    * @return void
+    */
+    public static function addReplyTo($replyTo, $replyToName):void
+    {
+        $this->replyTo = testEmail($replyTo, 'Reply-to')?$replyTo:false;
+        $this->replyToName = $replyToName;
+    }
+
+
+    /**
+     * Test for email address
+     * 
+     * @return bool
+     */
+    private static function testEmail(string $email, string $case):bool
+    {
+        if ($email && filter_var($email, FILTER_VALIDATE_EMAIL)) {
+            return true;
+        } else {
+            return false;
+            throw new Exception('Not valid "'.$case.'" email address!');
+        }
+    }
+
+
+    /**
+    * Send mail
+    *
+    * @return bool
+    */
+    public static function send():bool
+    {
+        if ($this->from && $this->fromName) {
+            $headders .= 'From: '.$this->fromName.' <'.$this->from.'>'."\r\n";
```

Comment:
It will be better to have another class only for headers.

Reply:
its just these now, really separate class?
![image](https://user-images.githubusercontent.com/47446443/105209348-04225b00-5b4a-11eb-9686-50ccf21ca409.png)

Reply:
It will be cleaner a and future proof. But, I can write it for you. 👍

Reply:
hold by beer